packages/neuron-wallet/src/database/chain/migrations/1652945662504-UpdateOutputChequeLockHash.ts

```diff
@@ -1,7 +1,7 @@
 import { In, MigrationInterface, QueryRunner } from "typeorm";
```

Comment:
maybe I should not edit this migration file, but it does not build if I don't change it, how should it be?

Reply:
I believe changing the type will not have a side effect.